feat(webview): add logging bridge and docs (#124)
enyst/OpenHands-Tab: docs/vscode_local_setup.md

>
> How to enable:
> - Automatic: launch the extension in Extension Development Host (Run/Debug) or during tests — logging is on by default.
> - Manual: open Settings and enable “OpenHands: Dev Bridge Enabled” (setting id: `openhands.devBridge.enabled`).
>
> What it does:
> - Captures webview console logs, unhandled errors, network requests and WebSocket lifecycle and forwards them to the extension.
> - Writes to Output: check the “OpenHands” output channel.
> - Writes to file (when enabled): `openhands-webview.log` inside the VS Code extension log folder.
>
> Where to find the log file:
> - Use the Command Palette: “Developer: Open Extension Logs Folder”, then open the `openhands.openhands-tab` folder and find `openhands-webview.log`.
> - Typical locations:
>   - macOS: `~/Library/Application Support/Code/logs/<timestamp>/exthost/openhands.openhands-tab/`
>   - Linux: `~/.config/Code/logs/<timestamp>/exthost/openhands.openhands-tab/`
>   - Windows: `%APPDATA%\Code\logs\<timestamp>\exthost\openhands.openhands-tab\`
>
> Production safety:

> > - The bridge is disabled by default for normal users; it only activates in Development/Test or if you opt-in via settings.

Author: ```suggestion
> - The bridge is disabled by default for normal/production use; it only activates in Development/Test or if you opt-in via settings.
```

> > - The bridge is disabled by default for normal/production use; it only activates in Development/Test or if you opt-in via settings.

Author: ```suggestion
```

  - Visual: Command Palette → Developer: Open Webview Developer Tools
  - CLI: Not reliably exposed via a stable command ID. Workaround: open Developer Tools (above) and select the webview iframe in the Elements panel.

Note on Webview DevTools from CLI
- Visual: Command Palette → Developer: Open Webview Developer Tools

> - CLI: Not reliably exposed via a stable command ID. Workarounds and alternatives:

Author: ```suggestion
- Alternatives:
```
